src/crab_utils/bignums_boost.hpp

```diff
@@ -22,7 +22,7 @@ class z_number final {
 
   public:
     z_number() = default;
-    explicit z_number(cpp_int n) : _n(std::move(n)) {}
+    z_number(cpp_int n) : _n(std::move(n)) {}
```

Comment:
Out of curiousity, can you explain this change? Did it cause a bad lookup? A warning?
(Of course it's an improvement, I just wonder what is the context here).

Reply:
It generated an error:

C:\git\dthaler\ebpf-verifier\src\crab_utils/bignums_boost.hpp(67,1): message :
Constructor for class 'crab::z_number' is declared 'explicit' [C:\git\dthaler\e
bpf-verifier\build\ebpfverifier.vcxproj]
C:\git\dthaler\ebpf-verifier\src\crab_utils/bignums_boost.hpp(72,49): error C24
40: '<function-style-cast>': cannot convert from 'boost::multiprecision::detail
::expression<boost::multiprecision::detail::multiply_immediates,boost::multipre
cision::number<boost::multiprecision::backends::cpp_int_backend<0,0,boost::mult
iprecision::signed_magnitude,boost::multiprecision::unchecked,std::allocator<bo
ost::multiprecision::limb_type>>,boost::multiprecision::et_on>,boost::multiprec
ision::number<boost::multiprecision::backends::cpp_int_backend<0,0,boost::multi
precision::signed_magnitude,boost::multiprecision::unchecked,std::allocator<boo
st::multiprecision::limb_type>>,boost::multiprecision::et_on>,void,void>' to 'c
rab::z_number' [C:\git\dthaler\ebpf-verifier\build\ebpfverifier.vcxproj]